CREDITS.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
Should this newline be included?

Reply:
I agree it shouldn't but I would fix it also in `main`